arches_querysets/models.py

```diff
@@ -249,7 +251,14 @@ def _save_aliased_data(
         )
         proxy_resource.save_descriptors()
         if index:
-            proxy_resource.index()
+            if (
+                hasattr(settings, "EVENTUALLY_CONSISTENT_ES_INDEXING")
+                and settings.EVENTUALLY_CONSISTENT_ES_INDEXING
+                and task_management.check_if_celery_available()
+            ):
```

Comment:
just a wee bit cleaner
```suggestion
            if (
                getattr(settings, "EVENTUALLY_CONSISTENT_ES_INDEXING", False)
                and task_management.check_if_celery_available()
            ):
```